File: app/core/departamentos.py
"""
Lista fija de departamentos de Bolivia usada por la Fundación.

Única fuente de verdad en el backend: valida `User.depto_asignado`
(RESPONSABLE_DEPARTAMENTAL) y el query param `depto` del router
`departmental`. El campo `Patient.depto` sigue siendo texto libre sin
validar (no se cambia ese comportamiento aquí), por eso el matching contra
esta lista se hace de forma tolerante (ver `app.core.text_normalize`).
"""

DEPARTAMENTOS = [
    "La Paz", "Cochabamba", "Santa Cruz", "Oruro",
    "Potosí", "Chuquisaca", "Tarija", "Beni", "Pando",
]

# Departamentos donde la Fundación asigna un RESPONSABLE_DEPARTAMENTAL.
# Pando queda excluido: no hay responsable en ese departamento. La lista
# completa (con Pando) se sigue usando para el filtro opcional del
# Coordinador Nacional y para no rechazar datos históricos de Patient.depto.
DEPARTAMENTOS_RESPONSABLE = [d for d in DEPARTAMENTOS if d != "Pando"]

File: vidaplena-web/src/constants/departamentos.js
// Lista fija de departamentos de Bolivia. Única fuente de verdad en el
// frontend — antes estaba duplicada en RegisterPatientPage y
// SelfRegisterPatientPage. También usada por UsersManagementPage para
// asignar departamento a un RESPONSABLE_DEPARTAMENTAL y por el panel
// departamental para filtrar (Coordinador Nacional).
export const DEPARTAMENTOS = [
  "La Paz", "Cochabamba", "Santa Cruz", "Oruro",
  "Potosí", "Chuquisaca", "Tarija", "Beni", "Pando",
];

// Departamentos donde la Fundación asigna un RESPONSABLE_DEPARTAMENTAL.
// Pando queda excluido (no hay responsable ahí). La lista completa se sigue
// usando para el filtro del Coordinador Nacional y el registro de pacientes.
export const DEPARTAMENTOS_RESPONSABLE = DEPARTAMENTOS.filter((d) => d !== "Pando");
